packages/typespec-test/test/todo_non_branded/generated/openapi/openapi.json

```diff
@@ -90,6 +90,7 @@
           }
         },
         "requestBody": {
+          "required": true,
```

Comment:
Should be a fix in OpenAPI emitter with shared route: https://github.com/microsoft/typespec/pull/4044